environment-pipelines/buildspec-apply.yml

```diff
@@ -17,17 +17,16 @@ phases:
       - echo -e "\nWorking on environment ${ENVIRONMENT}"
       - cd "terraform/environments/${ENVIRONMENT}"
       - terraform apply plan.tfplan
-      - echo -e "\nGenerating manifests and deploying AWS Copilot environment resources"
-      - cd "${CODEBUILD_SRC_DIR}"
   post_build:
     commands:
       - |
         if [ "${CODEBUILD_BUILD_SUCCEEDING}" == "1" ]
         then
           MESSAGE="Terraform apply phase complete"
         else
-          MESSAGE="Terraform apply phase FAILED"
+          MESSAGE=":alert: @here Terraform apply phase FAILED"
```

Comment:
I think this would need to be a standalone message, not a thread reply, as `@here` doesn't work in threads.

Reply:
Seemed to work in your tests earlier though, so lets see how it plays out in the wild.

Reply:
We've added in the --send-to-main-channel flag on the notify() command that alerts in the thread and the main channel that there's been a failure